client-app/src/Components/Login.js

```diff
@@ -54,67 +38,71 @@ const Login = props => {
     };
 
     return (
-        <div>
-            <form onSubmit={handleSubmit} className="my-5 container">
-                <div className="mb-3">
+        <div className="login-container">
+            <form onSubmit={handleSubmit} className="login-form">
+                <div className="form-group">
                     <label htmlFor="email" className="form-label">
                         Email address
                     </label>
                     <input
                         type="email"
                         className="form-control"
+                        name="email"
```

Comment:
Duplicate 'id' and 'name' attributes are present. Ensure each input has a unique 'id' if needed.